Misc/NEWS.d/next/Library/2018-02-16-14-37-14.bpo-32857.-XljAx.rst

```diff
@@ -0,0 +1 @@
+:mod:`tkinter` change handling of ``None`` parameter call to after_cancel.
```

Comment:
Could you please provide more useful information for readers? Like "after_cancel(None) now is error instead of canceling the first scheduled function."

And please add "Patch by *your name*."